osu.Game.Rulesets.Catch/Mods/CatchModFlashlight.cs

```diff
@@ -15,9 +16,30 @@ public class CatchModFlashlight : ModFlashlight<CatchHitObject>
     {
         public override double ScoreMultiplier => 1.12;
 
-        private const float default_flashlight_size = 350;
+        [SettingSource("Change radius based on combo", "Decrease the flashlight radius as combo increases.")]
+        public override BindableBool ChangeRadius { get; } = new BindableBool
+        {
+            Default = true,
+            Value = true
+        };
+
+        [SettingSource("Initial radius", "Initial radius of the flashlight area.")]
+        public override BindableNumber<float> InitialRadius { get; } = new BindableNumber<float>
+        {
+            MinValue = 0.4f,
+            MaxValue = 1.7f,
+            Default = 1f,
+            Value = 1f,
+            Precision = 0.1f
+        };
+
+        protected override BindableNumber<float> ModeMultiplier { get; } = new BindableNumber<float>
```

Comment:
Why is this a bindable?

Reply:
im confused, what should it be?

Reply:
Bindables are only used when the value is to be changed by a user or flow through the game. I'm not 100% sure if the overall direction here is correct, but this should just be a raw `float` not a bindable.

Reply:
i hope this suffices

Reply:
I'd also rename it to `FlashlightSize`.

Reply:
> I'd also rename it to `FlashlightSize`.

mm your call but idk, its not the flashlight size per say , its a per mode adjuster with which the final size is later obtained

Reply:
Then go with the original name. `DefaultFlashlightSize`.

Reply:
Done